shell/platform/linux/dart_platform.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define NDEBUG
+
+#include "flutter/shell/platform/linux/dart_platform.h"
+
+#include "dart/runtime/bin/platform.h"
+
+void InitDartPlatform(int argc, char* argv[]) {
+  dart::bin::Platform::SetExecutableName(argv[0]);
+  dart::bin::Platform::SetExecutableArguments(argc - 1, argv);
+}
```

Comment:
This code doesn't seem to have anything to do with linux.  Maybe move it into common?